sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/AssignedUser.cs

```diff
@@ -7,7 +7,7 @@
 
 using System;
 
-namespace Azure.ResourceManager.MachineLearningServices.Models
+namespace Azure.ResourceManager.MachineLearning.Models
 {
     /// <summary> A user that can be assigned to a compute instance. </summary>
     public partial class AssignedUser
```

Comment:
Are the two properties `Guid`?

Reply:
Probably. But I would like to defer this when we have more test cases. Now we have no test cases...